utils/post.js

```diff
@@ -82,7 +82,7 @@ const handleCreatePostTO = (
   return removeDuplicate;
 };
 
-const modifyPollPostObject = async (userId, item) => {
+const modifyPollPostObject = async (userId, item, isBlurredPost = true) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new parameter `isBlurredPost` is set to `true` by default. This could lead to unexpected behavior if the function is called without this argument, as it will blur posts even when not intended. Consider setting the default value to `false` to avoid unintentional blurring of posts.
```diff
- const modifyPollPostObject = async (userId, item, isBlurredPost = true) => {
+ const modifyPollPostObject = async (userId, item, isBlurredPost = false) => {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->